invenio_app_ils/records_relations/views.py

```diff
@@ -216,95 +222,111 @@ def _delete_sibling_relation(self, record, relation_type, payload):
     def _validate_sequence_creation_payload(self, record, payload):
         """Validate the payload when creating a new sequence relation."""
         try:
-            next_pid = payload.pop("next_pid")
+            next_pid_value = payload.pop("next_pid_value")
             next_pid_type = payload.pop("next_pid_type")
-            previous_pid = payload.pop("previous_pid")
+            previous_pid_value = payload.pop("previous_pid_value")
             previous_pid_type = payload.pop("previous_pid_type")
         except KeyError as key:
             raise RecordRelationsError(
                 "The `{}` is a required field".format(key)
             )
 
-        if record["pid"] != next_pid and record["pid"] != previous_pid:
+        if (
+            record["pid"] != next_pid_value
+            and record["pid"] != previous_pid_value
+        ):
             raise RecordRelationsError(
                 "Cannot create a relation for other record than one with PID "
                 "`{}`".format(record["pid"])
             )
 
-        if next_pid == previous_pid:
+        if next_pid_value == previous_pid_value:
             raise RecordRelationsError(
                 "Cannot create a sequence with the same next PID `{}`"
-                "and previous PID `{}`".format(next_pid, previous_pid)
+                "and previous PID `{}`".format(
+                    next_pid_value, previous_pid_value
+                )
             )
 
-        return next_pid, next_pid_type, previous_pid, previous_pid_type, \
-            payload
+        return (
+            next_pid_value,
+            next_pid_type,
+            previous_pid_value,
+            previous_pid_type,
+            payload,
+        )
 
     def _create_sequence_relation(self, record, relation_type, payload):
         """Create a Sequence relation.
 
         Expected payload:
 
             {
-                next_pid: <pid_value>,
+                next_pid_value: <pid_value>,
                 next_pid_type: <pid_type>,
-                previous_pid: <pid_value>,
+                previous_pid_value: <pid_value>,
                 previous_pid_type: <pid_type>,
-                relation_type: "<Relation name>",
+                relation_type: "<relation name>",
             }
         """
-        next_pid, next_pid_type, previous_pid, previous_pid_type, metadata = \
-            self._validate_sequence_creation_payload(record, payload)
+        next_pid_value, next_pid_type, previous_pid_value, previous_pid_type, metadata = self._validate_sequence_creation_payload(
```

Comment:
Also, too long line, doesn't pep complain in your tests? I will not comment any further long lines.